sdk/quantum/Azure.Quantum.Jobs/tests/Samples/QuantumJobClientSamples.HelloWorld.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.TestFramework;
+using Azure.Identity;
+using Azure.Quantum.Jobs.Models;
+using NUnit.Framework;
+
+namespace Azure.Quantum.Jobs.Tests.Samples
+{
+    public class QuantumJobClientSamples: SamplesBase<QuantumJobClientTestEnvironment>
+    {
+        [Test]
+        public void GettingASecret()
+        {
+//             var endpoint = TestEnvironment.KeyVaultUri;
+//
+//             #region Snippet:GetSecret
```

Comment:
I'm not sure if this will cause problems or not, but keeping the `GetSecret` snipped might anger the snippet generator.